.github/workflows/no-response.yml

```diff
@@ -0,0 +1,20 @@
+name: No Response
+
+# Both `issue_comment` and `scheduled` event types are required for this Action
+# to work properly.
+on:
+  issue_comment:
+    types: [created]
+  schedule:
+    # Schedule for five minutes after the hour, every hour
+    - cron: '5 * * * *'
```

Comment:
Maybe make it run every 12 hours (or even only once a day)? An interval of one hour between each run sounds unnecessary and wasteful to me.

Reply:
once a day is good enough. We just want to remind a few people to answer